tests/integration/dhcp_test.py

```diff
@@ -31,21 +33,53 @@
 IPV4_ADDRESS2 = '192.0.2.252'
 IPV6_ADDRESS1 = '2001:db8:1::1'
 IPV6_ADDRESS2 = '2001:db8:2::1'
+IPV4_CLASSLESS_ROUTE1 = '198.51.100.0/24'
+IPV4_CLASSLESS_ROUTE_DST1 = '192.0.2.1'
```

Comment:
It is not clear what this is exactly. Is this the gateway?

Reply:
No. It's classless route. 

Gateway is `0.0.0.0/2 via <gateway_ip>` when DHCP only provides gateway IP.
Classless route is `198.51.100.0/24 via 192.0.2.1` which require DHCP provides route and next-hop.

Reply:
Usually the format of any route entry is something like: <destination network> via <next hop>.

So perhaps use IPV4_DEST_NET and IPV4_NEXT_HOP or something similar.

Reply:
@EdDev's constant's names describe their purpose better, would be nice to change it.

Reply:
Changed to:
```
IPV4_CLASSLESS_ROUTE_DST_NET1 = '198.51.100.0/24'
IPV4_CLASSLESS_ROUTE_NEXT_HOP1 = '192.0.2.1'
IPV6_CLASSLESS_ROUTE_DST_NET1 = '2001:db8:f::/64'
```